.github/workflows/main.yml

```
@@ -125,7 +125,7 @@ jobs:
 
       - name: Publish package to PyPI
         run: |
-          twine upload -u ${{ secrets.PYPI_USERNAME }} -p ${{ secrets.PYPI_PASSWORD }} dist/*
+          twine upload -u ${{ secrets.PYPI_USERNAME }} -p '${{ secrets.PYPI_PASSWORD }}' dist/*
```

Might as well do so for the username too, right?

```suggestion
          twine upload -u '${{ secrets.PYPI_USERNAME }}' -p '${{ secrets.PYPI_PASSWORD }}' dist/*
```